modules/apps/static/portal-file-install/portal-file-install-test/src/testIntegration/java/com/liferay/portal/file/install/deploy/test/FileInstallConfigTest.java

```diff
@@ -144,8 +144,10 @@ public void testConfigurationArrayValues() throws Exception {
 
 	@Test
 	public void testConfigurationDeprecatedFileExtension() throws Exception {
-		Assume.assumeFalse(
-			PropsValues.MODULE_FRAMEWORK_FILE_INSTALL_CFG_ENABLED);
+		Boolean originalModuleFrameworkFileInstallCfgEnabled =
```

Comment:
@rafaprax @kyle-miho This may not work as java compiler may hard coded the constants value instead of referring to it.